docs/op_coverage.md

```diff
@@ -3,7 +3,7 @@ layout: default
 permalink: HLOOpCoverage
 title: XLA HLO Operation Coverage
 nav_order: 6
-js_files: 
```

Comment:
Remove local changes to this file? They'll be overwritten by the automated doc generation.

Reply:
VS Code auto-formatting can be a bit aggressive at times 😅